src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+          </th>
+
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            `consumer.offset`
+          </td>
+
+          <td>
+            The last consumed offset on a partition by the consumer group.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumer.lag`
+          </td>
+
+          <td>
+            The difference between a broker's high water mark and the consumer's offset (`consumer.hwm` - `consumer.offset`).
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumer.hwm`
+          </td>
+
+          <td>
+            The offset of the last message written to a partition (high water mark).
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumer.totalLag`
+          </td>
+
+          <td>
+            The sum of lags across partitions consumed by a consumer.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumerGroup.totalLag`
+          </td>
+
+          <td>
+            The sum of lags across all partitions consumed by a `consumerGroup`.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumerGroup.maxLag`
+          </td>
+
+          <td>
+            The maximum lag across all partitions consumed by a `consumerGroup`.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `consumerGroup.activeConsumers`
+          </td>
+
+          <td>
+            The number of active consumers in this `consumerGroup`.
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </Collapser>
+</CollapserGroup>
+
+## Configuration settings
+
+The following collapser contains all the configuration settings available:
+
+<CollapserGroup>
+    <Collapser
+        id="config-coll"
+        title="Configuration settings"
+    >
+## Configure KafkaBrokerSample and KafkaTopicSample collection [#broker-collection]
+
+The Kafka integration collects both Metrics and Inventory information. Check the **Applies To** column below to see the settings available to each collection:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **CLUSTER_NAME**
+      </td>
+
+      <td>
+        User-defined name to uniquely identify the cluster being monitored. **Required**.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **KAFKA_VERSION**
+      </td>
+
+      <td>
+        The version of the Kafka broker you're connecting to, used for setting optimum API versions. It must match -or be lower than- the version from the broker.
```

Comment:
Are the hyphens the best way to add emphasis here, or do we even need to emphasize this particularly?